src/components/card.tsx

```diff
@@ -9,9 +9,10 @@ type CitiesCardProp = {
   offer: OfferListItem;
   classType: 'city' | 'favorite' | 'offer';
   sizeImage: { width: number; height: number };
+  offerStateHandler: (offer?: OfferListItem) => void;
```

Comment:
когда обработчик передаем вниз в дочерний компонент то называем on + 
```suggestion
  onCardHover: (offer?: OfferListItem) => void;
```